opennlp-uima/src/main/java/opennlp/uima/normalizer/NumberUtil.java

```diff
@@ -90,12 +68,8 @@ public static Number parse(String number, String languageCode)
     }
 
     Locale locale = new Locale(languageCode);
-
     NumberFormat numberFormat = NumberFormat.getInstance(locale);
-
-    number = number.trim();
-    number = removeChar(number, ' ');
-
+    number = number.replaceAll("\\s", "");
```

Comment:
Possible follow up: https://issues.apache.org/jira/browse/OPENNLP-1441 (created in discussion with @mawiesne to improve performance throughout the code base)